python/lsst/dax/imgserv/locateImage.py

```diff
@@ -61,7 +55,31 @@ def image_open_v1(W13db, config, logger=log):
 
     """
     imagedb = W13db(config, logger)
-    return ImageGetter_v1(imagedb.butlerget, imagedb.metaservget, logger)
+    return ImageGetter(imagedb.butlerget, imagedb.metaservget, logger)
+
+
+def get_image(params, config) -> afw_image:
+    """Get the image per query request synchronously (default).
+    Parameters:
```

Comment:
This docstring doesn't follow any of our docstring standards.

Reply:
Fixed.